cloud_watch_alarm.tf

```diff
@@ -0,0 +1,38 @@
+# Restart dead or hung instance
+
+resource "null_resource" "check_alarm_action" {
+  count = "${local.instance_count}"
+
+  triggers = {
+    action = "arn:aws:swf:${local.region}:${data.aws_caller_identity.default.account_id}:${var.default_alarm_action}"
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "default" {
+  count               = "${local.instance_count}"
+  alarm_name          = "${module.label.id}"
+  comparison_operator = "${var.comparison_operator}"
+  evaluation_periods  = "${var.evaluation_periods}"
+  metric_name         = "${var.metric_name}"
+  namespace           = "${var.metric_namespace}"
+  period              = "${var.applying_period}"
+  statistic           = "${var.statistic_level}"
+  threshold           = "${var.metric_threshold}"
+  depends_on          = ["null_resource.check_alarm_action"]
+
+  dimensions {
+    InstanceId = "${aws_instance.default.id}"
+  }
+
+  alarm_actions = [
+    "${null_resource.check_alarm_action.triggers.action}",
+  ]
+}
+
+resource "null_resource" "eip" {
```

Comment:
This resource should be moved to `main.tf`.
It doesn't belong to CloudWatch assets.